vllm/engine/arg_utils.py

```diff
@@ -380,9 +381,12 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
                             action='store_true',
                             help='Disables sliding window, '
                             'capping to sliding window size')
-        parser.add_argument('--use-v2-block-manager',
-                            action='store_true',
-                            help='Use BlockSpaceMangerV2.')
+        parser.add_argument(
+            '--use-v2-block-manager',
+            default=EngineArgs.use_v2_block_manager,
+            action='store_true',
```

Comment:
It looks like a bug in upstream. If default is True, then action `store_true` will make the arg always be True i.e. there is no option to set it to False. Anyway, its on upstream, no changes to be done on fork,